flink-python/pyflink/table/expression.py

```diff
@@ -1533,6 +1533,8 @@ def json_value(self,
         ::
 
             >>> lit('{"a": true}').json_value('$.a') # STRING: 'true'
+            >>> lit("{"a.b": [0.998,0.996]}").jsonValue("$.['a.b'][0]", \
```

Comment:
This wouldn't compile.

```suggestion
            >>> lit('{"a.b": [0.998,0.996]}').json_value("$.['a.b'][0]", \
```